readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://konopliq.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://konopliq.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the pull request description 
<img width="925" height="289" alt="image" src="https://github.com/user-attachments/assets/039573fa-ed70-4b83-b5a4-6ed23a8d2323" />